InterfaceBuilder.md

```diff
@@ -6,8 +6,9 @@
 * Use `UIStackView`s instead of explicit constraints between siblings whenever possible, unless there are noticeable performance issues.
 * Each nib should have a single top level item.
     * Separate `UIView` subclasses designed in Interface Builder into their own nib files.
-* Use `IBDesignable` and `IBInspectable` for common design properties, e.g. specifying a view’s corner radius or giving it a border.
-* When setting colors in Interface Builder, use the ones defined in asset catalogs to ensure that all use of common colors is updated from a single source.
+* Use `IBInspectable` for common design properties, e.g. specifying a view’s corner radius or giving it a border. Avoid use of `IBDesignable` due to [outstanding tooling issues](http://www.openradar.appspot.com/radar?id=4952346488471552).
```

Comment:
Apple suggested during our call that this may not be a bug and could be working as intended. So I’d like to revise this to say use `IBDesginable` only for views with custom drawing (not to customize outlets) and use `IBInspectable` to customize properties of outlets.

Reply:
Please let me know if the update here matches your intention.